dev-docs/modules/userId.md

```diff
@@ -594,6 +594,11 @@ Please reach out to [FabrickIntegrations@team.neustar](mailto:FabrickIntegration
 | params.m | | String | This is a mobile advertising ID (IDFA/AAID) used to link a user to their Fabrick ID. | |
 | params.ia | | String | This is an identifier for advertising (IFA) used to link a user to their Fabrick ID. | |
 | params.iv | | String | This is an identifier for vendors (IFV) used to link a user to their Fabrick ID. | |
+| params.1pd | | String | This is the 1st party user ID (e.g. a Customer ID/CUSTID) - Note: Requires separate delivery of identity log files keyed off the 1st party user ID to establish an identity sync. | |
+| params.u | | String | This is the page_url - the url which the user is currently browsing. * encoding required for any character outside of alphabets (A-Z a-z), digits (0-9), hyphen (-), underscore (_) tilde (~), and dot (.). | |
+| params.f | | String | This is the referrer_url - the url which the user visited prior to landing on the page_url. * encoding required for any character outside of alphabets (A-Z a-z), digits (0-9), hyphen (-), underscore (_) tilde (~), and dot (.). | |
+| params.ifa_type | | String | This denotes the source of the IFA. Please refer to https://iabtechlab.com/wp-content/uploads/2018/12/OTT-IFA-guidelines.final_Dec2018.pdf for recommended values and additional details. | |
```

Comment:
Can you please change this to:
"Please refer to [IAB IFA Guidelines] (https://iabtechlab.com/wp-content/uploads/2018/12/OTT-IFA-guidelines.final_Dec2018.pdf) for recommended values and additional details."?

Reply:
Thank you for your input. I have submitted the update accordingly.